integration-cli/docker_cli_plugins_test.go

```diff
@@ -108,6 +108,16 @@ func (s *DockerSuite) TestPluginInstallDisable(c *check.C) {
 	c.Assert(strings.TrimSpace(out), checker.Contains, pName)
 }
 
+func (s *DockerSuite) TestPluginInstallDisableVolumeLs(c *check.C) {
+	testRequires(c, DaemonIsLinux, ExperimentalDaemon, Network)
+	out, _, err := dockerCmdWithError("plugin", "install", "--grant-all-permissions", "--disable", pName)
+	c.Assert(err, checker.IsNil)
+	c.Assert(strings.TrimSpace(out), checker.Contains, pName)
+
+	_, _, err = dockerCmdWithError("volume", "ls")
```

Comment:
@vieux yeah but you check that `err` is nil, that what `dockerCmd("volume", "ls")` basically does 👼